README.adoc

```diff
@@ -188,12 +208,12 @@ https://github.com/ChumbokIT[Github]
 
 |Evaly Limited
 |House #8, Road # 14, Dhanmondi, Dhaka-1209
```

Comment:
Hashtag in address should be removed, as GitHub renders #number as a pull request link.

Reply:
Fixed